src/Shape.java

```diff
@@ -1,5 +1,75 @@
-public  abstract class Shape {
-
-    abstract void draw(int a);//khai bao
-
-}
+public class Shape {
+    private double side1 = 1.0;
+    private double side2 = 1.0;
+    private double side3 = 1.0;
+    private double side4 = 1.0;
+    private String color = "black";
+
+    public Shape() {
+

```

Comment:
this.side1 = 1.0;
this.side2 = 1.0;
this.side3 = 1.0;
this.side4 = 1.0;
this.color = "black";